### PR TITLE
CSharpCollections_1 Hometask

### PR DESCRIPTION
**Pull request by NadzeyaKiz**
Hometask is completed

### CODE REVIEW

File: Code/CSharpCollections1/Task_01.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        {         
+            string[] numbers = input.Split(' ');
+
+            int[] array = new int[numbers.Length];
+            for (int i = 0; i < numbers.Length; i++)
+            {
+                if (!int.TryParse(numbers[i], out array[i]))
+                {
+                    Console.WriteLine($"Error! Invalid input: {input}");
+                    return new int[0];
```
Comment (MariaEroshevich) [inline Code/CSharpCollections1/Task_01.cs:27]: Do you really need to return value from massive here?

File: Code/CSharpCollections1/Task_02.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            double minNumber = array[0];
+            for (int i = 1; i < array.Length; i++)
+            {
+                if (array[i] < minNumber)
+                    minNumber = array[i];
+            }
+
+            return minNumber;
+        }
+        public double FindMaxNumber(double[] array)
```
Comment (MariaEroshevich) [inline Code/CSharpCollections1/Task_02.cs:32]: add space

**Review by MariaEroshevich — APPROVED** [review]

File: Code/CSharpCollections1/Task_01.cs (hunk 1 above)
Comment (NadzeyaKiz) [reply Code/CSharpCollections1/Task_01.cs:27]: In the provided code, the statement "return new int[0];" is used to handle the case where the input string cannot be parsed into an integer array. If the input string contains invalid or non-numeric characters, the parsing will fail for at least one element of the array. In this case,  to handle the error condition by returning an empty array.
By returning "new int[0]", the code indicates that an error has occurred during the parsing process. The calling code can check the returned array's length to determine if the parsing was successful or not. If the length is zero, it implies that there was an error, and the calling code can handle the error condition accordingly.
Returning an empty array in error conditions is just one way to handle errors. It is also possible to use throwing an exception in this case but I completed this before the "Exception" lesson.

File: Code/CSharpCollections1/Task_02.cs (hunk 2 above)
Comment (NadzeyaKiz) [reply Code/CSharpCollections1/Task_02.cs:32]: done

File: Code/CSharpCollections1/Task_01.cs (hunk 1 above)
Comment (MariaEroshevich) [reply Code/CSharpCollections1/Task_01.cs:27]: Good, I got it, thanks!

**Pull request merged**